hail_scripts/reference_data/combine.py

```diff
@@ -164,4 +165,5 @@ def update_existing_joined_hts(
         joined_ht = joined_ht.annotate(
             **{dataset: dataset_ht[joined_ht.locus][dataset]},
         )
-    return update_joined_ht_globals(joined_ht, dataset, version, genome_version)
+    joined_ht = update_joined_ht_globals(joined_ht, dataset, version, genome_version)
+    return joined_ht.checkpoint(checkpoint_path, overwrite=True, stage_locally=True)
```

Comment:
I've never tried returning a checkpoint like this. Does this actually checkpoint the table? It makes sense but I wouldn't be surprised if it writes but does not read it back in.

Reply:
Yep!  It does.  The docs actually say say it's  just an alias for `write` and then `read_table`:

https://hail.is/docs/0.2/_modules/hail/table.html#Table.checkpoint